payment/datastore.go

```diff
@@ -416,9 +418,8 @@ func (pg *Postgres) IsStripeSub(orderID uuid.UUID) (bool, string, error) {
 	return ok, "", err
 }
 
-// RenewOrder updates the orders status to paid and paid at time, inserts record of this order
-// 	Status should either be one of pending, paid, fulfilled, or canceled.
-func (pg *Postgres) RenewOrder(ctx context.Context, orderID uuid.UUID) error {
+// UpdateOrderExpiresAt - set the expires_at attribute of the order, based on now (or last paid_at if exists) and valid_for from db
```

Comment:
Out of curiosity, are we able to move the `UpdateOrderExpiresAt` block to `UpdateOrder`, and rename `UpdateOrderExpiresAt` to `updateOrderExpiresAt`?

Reply:
The main reason why I bring this up is that we're calling this from both the controller and a model, which isn't bad but the controller block is long.

Reply:
Agree.  This commit makes that a private method, and collapses it into update order: https://github.com/brave-intl/bat-go/pull/925/commits/2824cc408332cf3a040d65f3a4e60e2db091b21c  This change should resolve.